SoftLayer/managers/vs.py

```diff
@@ -368,10 +368,25 @@ def _generate_create_dict(
             data.update({
                 'primaryNetworkComponent': {
                     "networkVlan": {"id": int(public_vlan)}}})
+
         if private_vlan:
```

Comment:
My thoughts here are to break out the private/public vlan stuff into a helper function.


```python
if private_vlan or public_vlan:
    network_component = self._create_network_components(private_vlan, public_vlan, private_vlan_subnet, public_vlan_subnet)
    date.update(network_component)
```

Or something similar would work.

Reply:
Thanks for the suggestions, I had just started doing that very thing - however, just to pre-warn you it seems there is a tox check for max number of lines in a module, and we have just crossed the 1000 limit

Reply:
for that one, you can just add it to the ignored list at the top of this module. I don't want to split this file out at the moment, although I will likely do so in the future.